pkg/prometheus/server/statefulset.go

```diff
@@ -291,6 +291,18 @@ func makeStatefulSetSpec(
 	}
 	volumes, promVolumeMounts = appendServerVolumes(volumes, promVolumeMounts, queryLogFile, ruleConfigMapNames)
 
+	var watchedDirectories []string
+	configReloaderVolumeMounts := []v1.VolumeMount{
+		{
+			Name:      "config",
+			MountPath: prompkg.ConfDir,
+		},
+		{
+			Name:      "config-out",
+			MountPath: prompkg.ConfOutDir,
+		},
+	}
+
```

Comment:
nit: (same as previous nit) this change seems like it can be reverted, unless I'm missing something or if you believe it makes more sense have it in this part

Reply:
https://github.com/prometheus-operator/prometheus-operator/pull/5535#discussion_r1197680834